kratos/python/add_mesh_to_python.cpp

```diff
@@ -593,11 +700,19 @@ void  AddMeshToPython(pybind11::module& m)
     .def("CalculateOnIntegrationPoints", CalculateOnIntegrationPointsArray1d)
     .def("CalculateOnIntegrationPoints", CalculateOnIntegrationPointsVector)
     .def("CalculateOnIntegrationPoints", CalculateOnIntegrationPointsMatrix)
+    // GetValue  s  OnIntegrationPoints
```

Comment:
my proposal would be to rename the GetValueS... to CalculateValueS... and to also change it in the 3files that make use of it. 

Other than that i think we simply need to keep track of the change in 
https://github.com/KratosMultiphysics/Kratos/projects/26

and to mark the current PR as APIBreaker